### PR TITLE
Кальянов -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by KalyanovDO**

### CODE REVIEW

**Comment by VitKonevskiy** [comment]
@KalyanovDO , Ask your teammembers to review this PR. Thx.

**Comment by KalyanovDO** [comment]
@AndrewSmrnv, @VladimirTyagunov, please, review.

**Comment by AndrewSmrnv** [comment]
+1

**Comment by vtyagunov** [comment]
:+1:

**Comment by AndrewSmrnv** [comment]
:+1:

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        assertTrue(z1.equals(z2));
+    }
+
+    @Test
+    public void canCompareNonEquals() {
+        ComplexNumber z1 = new ComplexNumber(3, 5);
+        ComplexNumber z2 = new ComplexNumber(4, 6);
+        assertFalse(z1.equals(z2));
+    }
+
```
Comment (VitKonevskiy) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java]: Remove redundant line, please

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    @Test
+    public void getIm() {
+        ComplexNumber z1 = new ComplexNumber(342, 36483);
+        assertEquals(z1.getImaginary(), 36483, delta);
+    }
+
+    @Test
+    public void getRe() {
+        ComplexNumber z1 = new ComplexNumber(342, 36483);
+        assertEquals(z1.getReal(), 342, delta);
```
Comment (VitKonevskiy) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java]: Pls, check assertEquals function: 
`assertEquals(double expected, double actual, double delta)`
Could you swap arguments?

**Review by VitKonevskiy — CHANGES_REQUESTED** [review]

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java (hunk 1 above)
Comment (KalyanovDO) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java]: Thanks, issue was resolved.

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java (hunk 2 above)
Comment (KalyanovDO) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KalyanovDmitryTest.java]: Thanks, issue was resolved.

**Review by VitKonevskiy — APPROVED** [review]

**Pull request merged**